test/cypress/integration/simplified-account/service-settings/api-keys/api-keys.cy.js

```diff
@@ -169,6 +169,49 @@ describe('Settings - API keys', () => {
       })
     })
 
+    describe('revoke an api key', () => {
+      const TOKEN_LINK = 'token-link-2'
+      const DESCRIPTION = 'my api key'
+      const apiKeys = [
+        new Token().withCreatedBy('joe bloggs').withDescription(DESCRIPTION)
+          .withIssuedDate('10 Dec 2024').withLastUsed('10 Dec 2024').withTokenLink(TOKEN_LINK)
+      ]
+
+      beforeEach(() => {
+        setupStubs('admin', apiKeys)
+        cy.task('setupStubs', [
+          apiKeysStubs.getKeyByTokenLink(GATEWAY_ACCOUNT_ID, TOKEN_LINK, DESCRIPTION),
+          apiKeysStubs.revokeKey(GATEWAY_ACCOUNT_ID, TOKEN_LINK)
+        ])
+      })
+
+      it('should revoke the api key successfully', () => {
```

Comment:
maybe these should include something like "when 'Yes/No' is selected" - either in the test description or as a `describe` block just to make the expected behaviour clearer